plugin-server/src/main/services/http-server.ts

```diff
@@ -29,6 +29,12 @@ export function createHttpServer(hub: Hub, serverInstance: ServerInstance, serve
                 }
             }
 
+            // Unlike the above healthcheck, this is more of a "readiness" check that verifies that the consumer
+            // connected to the group successfully (thus being assigned a member id)
+            const mainConsumerHealthy = !serverInstance.queue || !!serverInstance.queue.consumerGroupMemberId
```

Comment:
So I think checking `consumerGroupMemberId` is a bad idea:
1. We have 0 tests for setting it
2. It isn't being unset when the consumer leaves the group

Let's create a proper boolean var and ensure it stays set correctly through the lifecycle?